### DISCUSSION TITLE
How to contribute component?

### DISCUSSION THREAD

**Discussion by yingshaoxo** [flet-dev/flet#1830] [General]
You know, if I use typescript, I could use npm to download others UI component, for example: 
* https://antdv.com/components/overview
* https://vant-ui.github.io/vant/mobile.html#/en-US

If I use flutter, I could download pub.dev UI package

But with `flet`, how can I reuse others package?

Should there have a name standard like `flet_PackageName`, so that we could install flet packages that was made by others directly from pypi.

**Comment by ndonkoHenri** [comment]
Have a look at this from the flet community: https://discord.com/channels/981374556059086931/1060276348263862343
